server/users.js

```diff
@@ -21,31 +19,116 @@ module.exports = require('express').Router()
     // forbidden('listing users is not allowed'),
     (req, res, next) =>
     User.findAll()
-    .then(users => res.json(users))
+    .then(users => res.status(200).json(users))
+    .catch(next))
+  // query to see non-admin users
+  .get('/?isAdmin="false"', (req, res, next) =>
+    User.findAll({where: {isAdmin: req.query.isAdmin}})
+    .then(users => res.status(200).json(users))
+    .catch(next))
+  // query to see if user is admin
+  .get('/?isAdmin="true"', (req, res, next) =>
+    User.findAll({where: {isAdmin: req.query.isAdmin}})
+    .then(users => res.status(200).json(users))
+    .catch(next))
+  // admin or actual user can view
+  .get('/:id', (req, res, next) =>
+    User.findById(req.params.id, {
+      include: [Review]
+    })
+    .then(user => res.json(user))
+    .catch(next))
+  // cant create a new user with admin status
```

Comment:
Kate said something about not just accepting req.body and specifying which fields you are receiving like req.body.name or req.body.email so that they can't just append anything they want to the body and ruin your database. Also since we can do something like making sure that isAdmin = false by setting it ourselves as a method of validation before doing a post or update